report_templates/host_-_system_currency.erb

```diff
@@ -0,0 +1,73 @@
+<%#
+name: Host - System currency
+snippet: false
+template_inputs:
+- name: Hosts filter
+  required: false
+  input_type: user
+  description: Limit the report only on hosts found by this search query. Keep empty
+    for report on all available hosts.
+  advanced: false
+  value_type: search
+  resource_type: Host
+- name: Errata filter
+  required: false
+  input_type: user
+  description: Limit the report only on errata found by this search query. Keep empty
+    for report on all available errata.
+  advanced: false
+model: ReportTemplate
+require:
+- plugin: katello
+￼ version: 3.9.0
+-%>
+<%  # multiplier for critical security errata -%>
+<%- sc_crit = 32 -%>
+<%  # multiplier for important security errata -%>
+<%- sc_imp = 16 -%>
+<%  # multiplier for moderate important security errata -%>
+<%- sc_mod = 8 -%>
+<%  # multiplier for low important security errata -%>
+<%- sc_low = 4 -%>
+<%  # multiplier for bugfix errrata -%>
+<%- sc_bug = 2 -%>
+<%  # multiplier for ehnancement errata -%>
+<%- sc_enh = 1 -%>
+<%- report_entries = [] -%>
+<%- load_hosts(search: input('Hosts filter'), includes: [:applicable_errata]).each_record do |host| -%>
+<%-   critical = 0 -%>
+<%-   important = 0 -%>
+<%-   moderate = 0 -%>
+<%-   low = 0 -%>
+<%-   bugfix = 0 -%>
+<%-   enhancement = 0 -%>
+<%-   host_applicable_errata_filtered(host, input('Errata filter')).each do |erratum| -%>
+<%-   critical+=1 if erratum.errata_type == 'security' and erratum.severity == 'Critical' -%>
+<%-   important+=1 if erratum.errata_type == 'security' and erratum.severity == 'Important' -%>
+<%-   moderate+=1 if erratum.errata_type == 'security' and erratum.severity == 'Moderate' -%>
+<%-   low+=1 if erratum.errata_type == 'security' and erratum.severity == 'Low' -%>
+<%-   bugfix+=1 if erratum.errata_type == 'bugfix' -%>
+<%-   enhancement+=1 if erratum.errata_type == 'enhancement' -%>
```

Comment:
wouldn't this deserver to live in `katello` itself? like `security_impact_score` with most of the impact scoring logic?
Or this is not interesting elsewhere than this report?

Reply:
Currently I don't see any other report in this repository or in katello repository using the same logic. I tried to search some reference about the `security_impact_score` that you mentioned.

Reply:
I don't believe there is some, but I think it could be beneficial somewhere else as well and I belive that the logic should be in Katello itself. It would look much cleaner. Thus I was suggesting to implement that method there :)